lib/parzival_web/router.ex

```diff
@@ -149,6 +149,7 @@ defmodule ParzivalWeb.Router do
     put "/settings", UserSettingsController, :update
     get "/settings/confirm_email/:token", UserSettingsController, :confirm_email
     get "/cv/:attendee_id", PdfController, :download_cv
+    post "/export_atteendes", CsvController, :create
```

Comment:
```suggestion
    get "/export_atteendes", CsvController, :create
```

This is for two reasons. One is if it is a GET you can access it just by pasting the URL into a browser. Second is for semantic reasons. You are not creating new information, you are simply accessing existing data.